### PR TITLE
Incorrect copy in mobile pay component #183 is Fixed

### PR DESCRIPTION
**Pull request by harith-hacky03**
Fixed the this.invoice in payment page

### CODE REVIEW

File: src/components/pages/bc-send-payment.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ export class SendPayment extends withTwind()(BitcoinConnectElement) {
                           block
                           @click=${this._copyAndDisplayInvoice}
                         >
-                          ${qrIcon}Copy & Display this.invoice
+                          ${qrIcon}Copy & Display
```
Comment (rolznz) [inline src/components/pages/bc-send-payment.ts:145]:
```suggestion
                          ${qrIcon}Copy & Display invoice
```

**Comment by rolznz** [comment]
Thanks @harith-hacky03 ! please see my suggestion.

**Comment by harith-hacky03** [comment]
Yeah Completed your suggestion too. Thank you @rolznz

**Comment by harith-hacky03** [comment]
Hope it helps and you will merge it . Thank you

**Comment by rolznz** [comment]
Thanks @harith-hacky03 !

**Pull request merged**